myproject/supermarket_manager/models.py

```diff
@@ -25,7 +26,9 @@ class Permission(models.Model):
     permission_id   = models.AutoField(primary_key=True)
     permission_name = models.CharField(max_length=100, unique=True)
     description     = models.CharField(max_length=100)
-    status          = models.CharField(max_length=1, choices=STATUS_CHOICES, default="A")
+    status = models.CharField(
+        max_length=10, choices=[(status.value, status.value) for status in AccountStatus], default=AccountStatus.ACTIVE.value
```

Comment:
Please update the name for `AccountStatus`, You are using them for many different models.

Reply:
Done